src/qn/qnitensor.jl

```diff
@@ -481,3 +484,17 @@ function splitblocks(A::ITensor, is=inds(A); tol=0)
   A = dropzeros(A; tol=tol)
   return A
 end
+
+function removeqn(T::ITensor, qn_name::String; mergeblocks=true)
+  if !hasqns(T)
+    return T
+  end
+  inds_R = removeqn(inds(T), qn_name; mergeblocks)
+  R = ITensor(inds_R)
+  for iv in eachindex(T)
+    if !iszero(T[iv])
+      R[iv] = T[iv]
+    end
+  end
```

Comment:
This is where the new tensor with the new storage is being made.

Reply:
Ok thanks! I kept missing it when looking through the changes. Yes later on I imagine we can have better facilities for slicing and then could do sliced assignments of entire blocks of T into either blocks or sub-blocks of R. (As a side thought, better slicing for both dense and block sparse tensors might simplify the block sparse combiner code.)

Reply:
Yeah, this would be a good candidate for making use of better slicing operations.